docs/process/testing.md

```diff
@@ -0,0 +1,232 @@
+### Download Desktop
+  - [ ] User can download latest (Mac & Windows) Desktop from https://desktop.github.com/    
+    - [ ] Mac: https://central.github.com/mac/latest 
```

Comment:
These are the installer URLs for the classic apps. I'd steal [the URLs from here](https://github.com/desktop/desktop#where-can-i-get-it) for the Electron apps.